<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reportes</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QTabWidget" name="tabsReportes">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>1015</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QScrollBar:vertical {
    width: 14px;
    background: transparent;
}

QScrollBar::handle:vertical {
    background: #8a8a8a;
    min-height: 80px;
    border-radius: 8px;
}

QScrollBar::handle:vertical:hover {
    background: #5f5f5f;
}

/* ocultamos flechas */
QScrollBar::add-line:vertical,
QScrollBar::sub-line:vertical {
    height: 0px;
}

/* üîë CLAVE: TRANSPARENT, NO NONE */
QScrollBar::add-page:vertical,
QScrollBar::sub-page:vertical {
    background: transparent;
}</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Stock de Productos</string>
    </attribute>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1011</width>
       <height>151</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1009</width>
        <height>149</height>
       </rect>
      </property>
      <widget class="QWidget" name="widget" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1011</width>
         <height>151</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QScrollBar:vertical {
    width: 24px;              /* üî• ancho de la barra */
    background: #e6e6e6;
}

QScrollBar::handle:vertical {
    background: #8a8a8a;
    min-height: 120px;         /* üî• tama√±o m√≠nimo del ‚Äúcoso‚Äù que se arrastra */
    border-radius: 8px;
}

QScrollBar::handle:vertical:hover {
    background: #5f5f5f;
}

QScrollBar::add-line:vertical,
QScrollBar::sub-line:vertical {
    height: 0px;
}

QScrollBar::add-page:vertical,
QScrollBar::sub-page:vertical {
    background: none;
}</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Productos m√°s vendidos</string>
    </attribute>
    <widget class="QScrollArea" name="scrollArea_productos_mas_vendidos">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1011</width>
       <height>151</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_productos_mas_vendidos">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1009</width>
        <height>149</height>
       </rect>
      </property>
      <widget class="QWidget" name="widget_2" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1011</width>
         <height>151</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QScrollBar:vertical {
    width: 24px;              /* üî• ancho de la barra */
    background: #e6e6e6;
}

QScrollBar::handle:vertical {
    background: #8a8a8a;
    min-height: 120px;         /* üî• tama√±o m√≠nimo del ‚Äúcoso‚Äù que se arrastra */
    border-radius: 8px;
}

QScrollBar::handle:vertical:hover {
    background: #5f5f5f;
}

QScrollBar::add-line:vertical,
QScrollBar::sub-line:vertical {
    height: 0px;
}

QScrollBar::add-page:vertical,
QScrollBar::sub-page:vertical {
    background: none;
}</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Por categor√≠a</string>
    </attribute>
    <widget class="QScrollArea" name="scrollArea_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1011</width>
       <height>151</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_categoria">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1009</width>
        <height>149</height>
       </rect>
      </property>
      <widget class="QWidget" name="widget_scroll_categoria" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1011</width>
         <height>141</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QScrollBar:vertical {
    width: 24px;              /* üî• ancho de la barra */
    background: #e6e6e6;
}

QScrollBar::handle:vertical {
    background: #8a8a8a;
    min-height: 120px;         /* üî• tama√±o m√≠nimo del ‚Äúcoso‚Äù que se arrastra */
    border-radius: 8px;
}

QScrollBar::handle:vertical:hover {
    background: #5f5f5f;
}

QScrollBar::add-line:vertical,
QScrollBar::sub-line:vertical {
    height: 0px;
}

QScrollBar::add-page:vertical,
QScrollBar::sub-page:vertical {
    background: none;
}</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Ventas Mensuales</string>
    </attribute>
    <widget class="QScrollArea" name="scrollArea_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1011</width>
       <height>151</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_ventas">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1009</width>
        <height>149</height>
       </rect>
      </property>
      <widget class="QWidget" name="widget_scroll_ventas" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1011</width>
         <height>141</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QScrollBar:vertical {
    width: 24px;              /* üî• ancho de la barra */
    background: #e6e6e6;
}

QScrollBar::handle:vertical {
    background: #8a8a8a;
    min-height: 120px;         /* üî• tama√±o m√≠nimo del ‚Äúcoso‚Äù que se arrastra */
    border-radius: 8px;
}

QScrollBar::handle:vertical:hover {
    background: #5f5f5f;
}

QScrollBar::add-line:vertical,
QScrollBar::sub-line:vertical {
    height: 0px;
}

QScrollBar::add-page:vertical,
QScrollBar::sub-page:vertical {
    background: none;
}</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QTableWidget" name="tabla_reportes">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>1011</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <column>
    <property name="text">
     <string>Fecha</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Cantidad de ventas</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Total del d√≠a</string>
    </property>
   </column>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>381</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="buscador">
    <item>
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>Desde:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="fechaDesde">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>75</width>
        <height>20</height>
       </size>
      </property>
      <property name="displayFormat">
       <string>d/MM/yyyy</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>Hasta:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="fechaHasta">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>75</width>
        <height>20</height>
       </size>
      </property>
      <property name="displayFormat">
       <string>d/MM/yyyy</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="botonBuscar">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>75</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>Buscar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_volver">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>32</height>
    </size>
   </property>
   <property name="font">
    <font>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    padding: 4px 10px;
    font-size: 11px;
    background: transparent;
    border: none;
}

QPushButton:hover {
    color: #2e7d32;  
}</string>
   </property>
   <property name="text">
    <string>Volver</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../resources/icons/back.png</normaloff>../../resources/icons/back.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_imprimir">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>410</y>
     <width>200</width>
     <height>39</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>49</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #4CAF50;
    color: white;
    font-size: 16px;
    padding: 10px 20px;
    border-radius: 5px;
}</string>
   </property>
   <property name="text">
    <string>Imprimir</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>0</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;Reportes&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
